vms/evm/subscriber.go

```diff
@@ -75,16 +121,89 @@ func NewWarpLogInfo(log types.Log) (*vmtypes.WarpLogInfo, error) {
 // forward logs from the concrete log channel to the interface channel
 func (s *subscriber) forwardLogs() {
 	for msgLog := range s.evmLog {
-		messageInfo, err := NewWarpLogInfo(msgLog)
+		messageInfo, err := s.NewWarpLogInfo(msgLog)
 		if err != nil {
-			s.logger.Info(
+			s.logger.Error(
 				"Invalid log. Continuing.",
 				zap.Error(err),
 			)
 			continue
 		}
 		s.log <- *messageInfo
+
+		// Update the database with the latest seen block height
```

Comment:
Is it correct that we may update the seen block height here prior to all the logs in the block being processed?

Reply:
Yes. This is desired as described here: https://github.com/ava-labs/awm-relayer/pull/20#discussion_r1316422605